docker/build_scripts/build-cpython.sh

```diff
@@ -39,16 +39,22 @@ if [ "${CPYTHON_VERSION}" == "3.6.15" ]; then
 	CFLAGS_EXTRA="${CFLAGS_EXTRA} -fno-tree-loop-vectorize -fno-tree-slp-vectorize"
 fi
 if [ "${AUDITWHEEL_POLICY}" == "manylinux2014" ] ; then
-    # Python 3.11+
+	# Python 3.11+
 	export TCLTK_LIBS="-ltk8.6 -ltcl8.6"
 fi
+if [ "${CPYTHON_VERSION%.*}" == "3.6" ] || [ "${CPYTHON_VERSION%.*}" == "3.7" ]; then
+	OPENSSL_OPTIONS=""
+else
+	OPENSSL_OPTIONS="--with-openssl=/usr/local/openssl3"
```

Comment:
This won't work if the system openssl is good enough as /usr/local/openssl3 won't exist.
Am I missing something ?

Reply:
Fixed.